Flow.Launcher.Plugin/IPublicAPI.cs

```diff
@@ -34,6 +34,11 @@ public interface IPublicAPI
         /// </summary>
         void SaveAppAllSettings();
 
+        /// <summary>
+        /// Save Flow's plugins settings
+        /// </summary>
+        void SavePluginSettings();
```

Comment:
what's the difference between this method and SaveSettingJsonStorage()?

Reply:
SaveSettingJsonStorage is for a specific Type (or T as generic), but this is for all plugin.